adapter/README.md

```diff
@@ -4,6 +4,22 @@
 
 An adapter to easily run an [Axum](https://github.com/tokio-rs/axum) server in a Cloudflare worker.
 
+## Cloudflare workers perliminary native support for Axum on 0.0.21+
+
+Its hidden behind the feature flag "http".
+
+This package can be used as an easy way to migrate from the non http version to the http version:
```

Comment:
```suggestion
This package can be used as an easy way to migrate from the non http worker-rs version to the http version:
```

Reply:
Picking your nit pick: the crate is called `worker` but the repo is called `workers-rs` (plural).

Reply:
Thank you, I've been referencing it the wrong the way for a long time! 😳